### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by zrybeauty** [Goluck-Konuko/hdac#3]
Hi, could you please share the training set (processed VoxCeleb2) and test set (processed VoxCeleb2 and Xiph.org)? Thank you so much.

**Comment by zrybeauty** [comment]
I found that.
